_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Picture of Jaime](https://user-images.githubusercontent.com/109662648/181019998-8661e7e0-0688-4b4e-85b2-4e34b132318f.jpg)